### PR TITLE
golang vendor dependencys once to not timeout

### PR DESCRIPTION
**Pull request by 6543**
dont have timeouts like https://ci.woodpecker-ci.org/repos/6032/pipeline/975/5

### CODE REVIEW

**Comment by 6543** [comment]
YES

**Comment by 6543** [comment]
that's why we get timeouts ....

**Comment by 6543** [comment]
from ~25min down to 2

File: .woodpecker/test.yml [hunk 1]
```diff
@@ -7,20 +7,27 @@ when:
...
   test:
     image: *golang
+    depends_on: [ vendor ]
     commands:
       - make test
 
   build:
     image: woodpeckerci/plugin-docker-buildx:4.0.0
-    depends_on: [ lint, test ]
+    depends_on: [ lint, test, vendor ]
```
Comment (lafriks) [inline .woodpecker/test.yml:30]: This does not seem needed as lint and test already depend no vendor

Comment (6543) [reply .woodpecker/test.yml:30]: yes but if in future we alter stuff ... that step do explizit need vendor too ... here it is a noop but the documentation porpuse stayews

**Comment by 6543** [comment]
> OK, didn't see this. But it's actually weird, shouldn't it cache them? Or is the problem that this doesn't work when building multiarch docker containers?

well 
https://github.com/woodpecker-ci/plugin-s3/blob/0903b1fb25dbeee18443d64731624ab2d662f1b8/Dockerfile.multiarch#L6-L8

should cache to some extend ... but it looks like it does not work in our case ...?!?

**Pull request merged**